webrtc/RTCPeerConnection.d.ts

```diff
@@ -121,33 +118,19 @@ interface RTCDataChannelInit {
 }
 
 // TODO(1)
-declare enum RTCSdpType {
-  // http://dev.w3.org/2011/webrtc/editor/webrtc.html#rtcsdptype
-  'offer',
-  'pranswer',
-  'answer'
-}
+type RTCSdpType = 'offer' | 'pranswer' | 'answer' | 'rollback';
 
 interface RTCMessageEvent {
-  // http://dev.w3.org/2011/webrtc/editor/webrtc.html#event-datachannel-message
-  // At present, this can be an: ArrayBuffer, a string, or a Blob.
-  // See TODO(2)
-  data: any;
+  data: ArrayBuffer | string | Blob;
```

Comment:
I vote 👍  for `any`.
https://www.w3.org/TR/webmessaging/#the-messageevent-interfaces